$NetBSD: patch-auto_shmem,v 1.1 2021/12/13 23:10:22 gutteridge Exp $

Fix sandboxed builds on NetBSD, where /var/shm typically doesn't exist.

--- auto/shmem.orig	2025-02-27 22:02:21 UTC
+++ auto/shmem
@@ -9,7 +9,11 @@ NXT_SHM_PREFIX="/"
 
 nxt_feature="shm_open()"
 nxt_feature_name=NXT_HAVE_SHM_OPEN
+if [ $NXT_SYSTEM = NetBSD ]; then
+nxt_feature_run=no
+else
 nxt_feature_run=yes
+fi
 nxt_feature_incs=
 nxt_feature_libs=
 nxt_feature_test="#include <sys/mman.h>
